samples/bluetooth/peripheral_gatt_dm/README.rst

```diff
@@ -0,0 +1,63 @@
+.. _peripheral_gatt_dm:
+
+Bluetooth: Peripheral GATT Discovery Manager
+############################################
+
+The peripheral GATT Discovery sample demonstrates how to use the :ref:`gatt_dm_readme`.
+
+Overview
+********
+
+When connected, the sample discovers services of the connected device.
+
+Requirements
+************
+
+* One of the Nordic development boards:
+
+  * nRF52840 Development Kit board (PCA10056)
+  * nRF52 Development Kit board (PCA10040)
+  * nRF51 Development Kit board (PCA10028)
+
+* A device to connect to the peripheral, for example a phone or a tablet with `nRF Connect for Mobile`_, or `nRF Toolbox`_.
+
+Building and Running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_gatt_dm`
+
+.. include:: /includes/build_and_run.txt
+
+.. _peripheral_gatt_dm_testing:
+
+Testing
+=======
+
+After programming the sample to your dongle or development board, test it by performing the following steps.
+This testing procedure assumes that you are using `nRF Connect for Mobile`_.
+
+1. Connect the board to the computer using a USB cable.
+   The board is assigned a COM port (Windows) or ttyACM device (Linux), which is visible in the Device Manager.
+#. Connect to the board with a terminal emulator, for example, LTE Link Monitor.
```

Comment:
```suggestion
#. |connect_terminal|
```

Reply:
This will resolve to "Connect to the board with a terminal emulator (for example, PuTTY). See :ref:`putty` for the required settings."
Or is there a specific reason to recommend LTE Link Monitor here?